package.json

```diff
@@ -7,9 +7,10 @@
   "private": true,
   "dependencies": {
     "@audius/libs": "1.2.13",
-    "@audius/stems": "0.3.12",
+    "@audius/stems": "^0.3.13",
```

Comment:
intentional?

Reply:
Ideally a patch version update wouldn't break us, but I'll change it, thx